packages/renderer/src/lib/dialogs/CommandPaletteUtils.ts

```
@@ -0,0 +1,81 @@
+
+export function getGoToDisplayText(goToInfo: GoToInfo): string {
+  if (goToInfo.kind === 'Image') {
+    const imageInfo = goToInfo.info as ImageInfo;
+    return getShortId(imageInfo.RepoTags?.[0] ?? imageInfo.Id);
+  } else if (goToInfo.kind === 'Container') {
+    return (goToInfo.info as ContainerInfo).Names[0];
+  } else if (goToInfo.kind === 'Pod') {
+    return (goToInfo.info as PodInfo).Name;
+  } else if (goToInfo.kind === 'Volume') {
+    return (goToInfo.info as VolumeInfo).Name.substring(0, 12);
+  }
```

Biome (2.1.2)

[error] This else clause can be omitted because previous branches break early.

Safe fix: Omit the else clause.


(lint/style/noUselessElse)